source/_integrations/speedtestdotnet.markdown

```diff
@@ -22,9 +22,7 @@ The Speedtest.net integration uses the [Speedtest.net](https://speedtest.net/) w
 
 Most Speedtest.net servers require TCP port 8080 outbound to function. Without this port open you may experience significant delays or no results at all. See note on their [help page](https://www.speedtest.net/help).
 
-## Manual Polling
-
-By default, a speed test will be run every hour. To customize polling, you can disable automated polling. Go to {% my integrations title="**Settings** > **Devices & Services**" %}, and select the **Speedtest** integration. On the integration entry, select the three dots. Then, select **System options** and toggle the button to disable polling. To manually trigger a polling request, call the [`homeassistant.update_entity` service](/integrations/homeassistant/#service-homeassistantupdate_entity) as needed, either manually or via automations.
```

Comment:
Should we create an item like you did for enabling entities? And reference that instead?

Like `{% include common-tasks/enable_entities.md %}` (which is also included on the independent common tasks lists).